### PR TITLE
Bug: Fix bug - fix bug of hipBusBandwidth build

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
fix bug of hipBusBandwidth building

**Major Revision**
- it failed to enter the check 'hip/samples/1_Utils/hipBusBandwidth/CMakeLists.txt' when building docker, so removed this check
- add sb_micro_path for rocm_bandwidthTest

### CODE REVIEW

File: third_party/Makefile [hunk 1]
```diff
@@ -93,9 +93,7 @@ endif
 # Build hipBusBandwidth.
 # HIP is released with rocm, like rocm-4.2.0 and so on.
 # The version we use is the released tag which is consistent with the rocm version in the environment or docker.
-rocm_bandwidthTest:
+rocm_bandwidthTest: sb_micro_path
 	cp -r -v $(shell hipconfig -p) ./
-ifneq (, $(wildcard hip/samples/1_Utils/hipBusBandwidth/CMakeLists.txt))
```
Comment (abuccts) [inline third_party/Makefile:98]: can you copy the `hipBusBandwidth/` directory only?

**Review by abuccts — APPROVED** [review]

File: third_party/Makefile (hunk 1 above)
Comment (yukirora) [reply third_party/Makefile]: ok, sure, revised

**Review by cp5555 — APPROVED** [review]

File: third_party/Makefile [hunk 2]
```diff
@@ -93,9 +93,7 @@ endif
...
 # The version we use is the released tag which is consistent with the rocm version in the environment or docker.
-rocm_bandwidthTest:
-	cp -r -v $(shell hipconfig -p) ./
-ifneq (, $(wildcard hip/samples/1_Utils/hipBusBandwidth/CMakeLists.txt))
-	cd ./hip/samples/1_Utils/hipBusBandwidth/ && mkdir -p build && cd build && cmake .. && make
-	cp -v ./hip/samples/1_Utils/hipBusBandwidth/build/hipBusBandwidth $(SB_MICRO_PATH)/bin/
-endif
+rocm_bandwidthTest: sb_micro_path
+	cp -r -v $(shell hipconfig -p)/samples/1_Utils/hipBusBandwidth ./
+	cd ./hipBusBandwidth/ && mkdir -p build && cd build && cmake .. && make
```
Comment (guoshzhao) [inline third_party/Makefile:98]: should delete `./hipBusBandwidth/`?

Comment (yukirora) [reply third_party/Makefile:98]: why? I copy the directory 'hipBusBandwidth' to current path, and cd hipBusBandwidth to build

Comment (guoshzhao) [reply third_party/Makefile:98]: After build, the `./hipBusBandwidth/` is not needed any more?

File: third_party/Makefile [hunk 3]
```diff
@@ -93,9 +93,8 @@ endif
...
+	cp -v ./hipBusBandwidth/build/hipBusBandwidth $(SB_MICRO_PATH)/bin/
+	rm -rf hipBusBandwidth
```
Comment (abuccts) [inline third_party/Makefile:100]: why only remove this directory but not for others?
you can add `make clean` if you want to delete the build directory

**Review by guoshzhao — APPROVED** [review]

**Pull request merged**